### DISCUSSION TITLE
Issue with setting up a storybook.

### DISCUSSION THREAD

**Discussion by BorisSokachev** [storybookjs/storybook#24802] [Help] [answered]
Hi,

I'm following they youtube tutorial series on Chromatic's channel and I immediately ran into my first problem. 

Im super new at this. 

When I try to validate a storybook in VSC I get this error and I don't know how to continue.
<img width="633" alt="Screenshot 2023-11-10 at 12 44 13" src="https://github.com/storybookjs/storybook/assets/134837470/16e10a61-c57f-456e-a669-4539245a0e1b">

The tutorial in question:
<img width="984" alt="Screenshot 2023-11-10 at 12 42 57" src="https://github.com/storybookjs/storybook/assets/134837470/2e6602d8-d316-4220-b6fb-5baadb9c0037">

**Comment by valentinpalkovic** [comment, marked as answer]
Hi @BorisSokachev,

I think you execute the command in your home directory (`~`). You have to go to the folder, where Storybook was initiated

**Reply by BorisSokachev** [reply]
I see. That brings up the question of how would I initiate Storybook.

**Reply by valentinpalkovic** [reply]
We are not supporting setting up Storybook in an empty directory. If you have an app in place (Next.js, Angular, Svelte, Vue,...), go to that project directory and instantiate Storybook on top of it: https://storybook.js.org/docs/react/get-started/install

**Reply by BorisSokachev** [reply]
Thank you this answered my questions outstandingly!
